<launch>
    <param name="dataset_folder" type="string" value="/home/gyuseok/Downloads/2011_10_03_drive_0027_extract/2011_10_03/2011_10_03_drive_0027_extract/velodyne_points/data/" />
    <param name="publish_delay" type="int" value="10" />
    <param name="load_frame_num" type="int" value="20" />
    <param name="sequence_number" type="string" value="00" />
    <param name="to_bag" type="bool" value="false" />
    <param name="load_gt_path" type="bool" value="false" />
    <param name="output_bag_file" type="string" value="/home/gyuseok/catkin_ws_kitti/test.bag" />
    
    <node name="kittiHelperGyu1" pkg="aloam_velodyne" type="kittiHelper" output="screen" />

</launch>